android/src/main/java/com/cloudwebrtc/webrtc/GetUserMediaImpl.java

```diff
@@ -751,6 +751,13 @@ private ConstraintsMap getUserVideo(ConstraintsMap constraints, MediaStream medi
         deviceId = result.first;
         VideoCapturer videoCapturer = result.second;
 
+        if (facingMode == null && cameraEnumerator.isFrontFacing(deviceId)) {
```

Comment:
I'm not sure if we should check `facingMode == null` here and below. Is it possible to set a facingMode constraint and get a camera that doesn't fulfil it?